app/data_models/app_models.py

```diff
@@ -56,8 +75,13 @@ class DateTimeSchemaMixin:
     created_at = fields.DateTime()
     updated_at = fields.DateTime()
 
+    # pylint: disable=unused-argument
```

Comment:
Is this function better off as a `@staticmethod` then?

Reply:
I've made it static since `self` was not needed, extra test added as well, thanks!